.buildkite/scripts/build_kibana.sh

```diff
@@ -12,13 +12,19 @@ else
 fi
 
 if [[ "${GITHUB_PR_LABELS:-}" == *"ci:deploy-cloud"* ]]; then
-  echo "--- Build Kibana Cloud Distribution"
+  echo "--- Build and push Kibana Cloud Distribution"
+
+  echo "$KIBANA_DOCKER_PASSWORD" | docker login -u "$KIBANA_DOCKER_USERNAME" --password-stdin docker.elastic.co
+  trap 'docker logout docker.elastic.co' EXIT
+
   node scripts/build \
     --skip-initialize \
     --skip-generic-folders \
     --skip-platform-folders \
     --skip-archives \
     --docker-images \
+    --docker-tag-qualifier="-$GIT_COMMIT" \
```

Comment:
We should update the qualifier to no have to supply the hyphen

Reply:
Pushed with 9634ed3e7d467387d6e43e04293a1c741ba7a97d